wifi-connect-exclave/logic/scripts/cli.bash

```diff
@@ -3,7 +3,7 @@ echo "Running test-01-cli..."
 # Clone the wifi connect repo (bash script)
 
 echo "git cloning..."
-git clone https://github.com/balena-io/wifi-connect.git ~/wifi-connect && cd ~/wifi-connect || exit 1
+git clone https://github.com/balena-io/wifi-connect.git ~/wifi-connect && cd ~/wifi-connect
```

Comment:
If the folder is already there, then the script exits out. Hence, remove exit 1

Reply:
@vipulgupta2048 another thing that happens - if wifi connect is already cloned, then it doesn't do the cd into the directory - so when you get down to balena push, it doesn't push from the right place.